doc/api/fs.md

```diff
@@ -4719,6 +4719,69 @@ The following constants are meant for use with the [`fs.Stats`][] object's
   </tr>
 </table>
 
+### File System Flags
+
+The following flags are available wherever the `flag` option takes a
+string:
+
+* `'r'` - Open file for reading.
```

Comment:
Can you please sort flags in alphabetical order?

Reply:
ok, I'm on it!
Would it also make sense to rearrange the existing flag on open() to also be rearranged, or is there a reason to keep the other ones in their original order?

Reply:
I'll let @nodejs/fs take a call, in `fs.open()` the flags are separated into read and write ones.

I don't think the description for `flags` should be repeated in the documentation. In my opinion, `fs.open()` should also link to this description for flags.

Reply:
I think sorting the flags is a good idea.